mobile/about.html

```diff
@@ -12,6 +12,48 @@ <h1>About this document</h1>
     <main data-contents="about">
       <section>
         <h2>Change History</h2>
+        <section>
+          <h3>September 2020</h3>
+          <p>The <a href="https://www.w3.org/2020/09/web-roadmaps/mobile/">September 2020</a> snapshot adds a few new proposals being incubated in various W3C community groups, some new deliverables from various W3C working groups, and refreshes the status of other specifications. Changes made since the November 2019 snapshot:</p>
```

Comment:
Juste propose to streamline this to "The September 2020 snapshot adds new W3C community groups incubations, new W3C working group deliverables, and refreshes the status of other specifications."

Reply:
Fixed. Thank you.